src/generated/xinput.rs

```diff
@@ -4305,8 +4246,7 @@ where
         .ok_or(ConnectionError::UnsupportedExtension)?;
     let length_so_far = 0;
     let device_id_bytes = device_id.serialize();
-    assert_eq!(0, keymaps.len() % 8, "Argument keycodes_per_modifier has an incorrect length, must be a multiple of 8");
-    let keycodes_per_modifier = u8::try_from(keymaps.len() / 8).unwrap();
+    let keycodes_per_modifier = keymaps.len() as u8 / 8;
```

Comment:
This assert is needed to generate a correct request.

The `try_from` is also a good idea, I think. Truncation would lead to an incorrect request being generated.

Reply:
Fixed